src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -581,13 +581,12 @@ validate_args(Args) ->
             ok;
         {[], _, _} ->
             ok;
+        {[Key], StartKey, EndKey} when Key =:= StartKey andalso Key =:= EndKey ->
```

Comment:
`{[Key], Key, Key}` is a less verbose equivalent. Not essential.

Reply:
Good idea!
Changed, thanks for the review!